src/make/Make.include

```diff
@@ -492,6 +506,7 @@ CPP_INCLUDES := \
     $(ROOT_INCLUDES) \
     $(PYTHIA8_INCLUDES) \
     $(LHAPDF_INCLUDES) \
+    $(APFEL_INCLUDES) \
```

Comment:
I suggest to include the APFEL libraries only when required in the specific makefiles. not at the top level. Examples can be seen in [Robet's code](https://github.com/nusense/Generator/blob/inclxx_and_g4/src/Physics/HadronTransport/Makefile) where we include INCL++ and Geant4 as specific packages